fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -177,7 +161,16 @@ export class EfisSymbols<T extends number> {
                 continue;
             }
 
-            if (mode === EfisNdMode.PLAN && !planCentre) {
+            const termination = this.findPlanCentreCoordinates(side);
+            if (termination) {
+                SimVar.SetSimVarValue(`L:A32NX_EFIS_${side}_MRP_LAT`, 'Degrees', termination.lat);
+                SimVar.SetSimVarValue(`L:A32NX_EFIS_${side}_MRP_LONG`, 'Degrees', termination.long);
+            } else {
+                SimVar.SetSimVarValue(`L:A32NX_EFIS_${side}_MRP_LAT`, 'Degrees', ppos.lat);
+                SimVar.SetSimVarValue(`L:A32NX_EFIS_${side}_MRP_LONG`, 'Degrees', ppos.long);
+            }
```

Comment:
I would suggest using subjects and suitable quantisation to avoid writing these every frame.